tables-test-fixtures/tables-test-fixtures-iceberg-1.2/src/main/java/com/linkedin/openhouse/tablestest/TestSparkSessionUtil.java

```diff
@@ -37,7 +37,7 @@ public static void configureCatalogs(
     builder
         .config(
             String.format("spark.sql.catalog.%s", catalogName),
-            "org.apache.iceberg.spark.SparkCatalog")
+            "com.linkedin.openhouse.spark.SparkCatalog")
```

Comment:
We'd need to change Docker code too, and all other references to this connector.